main/FatesHistoryInterfaceMod.F90

```diff
@@ -3612,7 +3612,7 @@ subroutine define_history_vars(this, initialize_variables)
          avgflag='A', vtype=patch_r8, hlms='CLM:ALM', flushval=0.0_r8, upfreq=1,   &
          ivar=ivar, initialize=initialize_variables, index = ih_fire_fuel_mef_pa )
 
-    call this%set_history_var(vname='fire_fuel_bulkd', units='m',              &
+    call this%set_history_var(vname='fire_fuel_bulkd', units='kgbiomass/m2',              &
```

Comment:
is this kg biomass or kg carbon?

Reply:
@ckoven it is kgbiomass.

Reply:
@ckoven there may be an issue in the code where fire intensity is calculated. W is a conversion from TFC_ROS. TFC_ROS is divided by 0.45 to convert it to kg biomass. From what I can tell TFC_ROS is already in kg biomass and  does not to be converted. I want to look through this again to confirm, but will likely remove that divide by 0.45 for the calculation of W from TFC_ROS in fire_intensity. Looks like a mistake.

Reply:
Added a correction here to kg biomass/m3